uportal-war/src/main/java/org/apereo/portal/portlets/backgroundpreference/RoleBasedBackgroundSetSelectionStrategy.java

```diff
@@ -78,6 +78,10 @@ public String getSelectedBackgroundImagePreferenceName() {
         public String getBackgroundContainerSelectorPreferenceName() {
             return prefix + "BackgroundContainerSelector";
         }
+
+        public String getImageCaptionsName() {
+            return "backgroundImageCaptions";
```

Comment:
I have this change, but I'm curious - would it be better to match the style of the surrounding code in the PreferenceNames enum? Putting a static field in with four other methods feels inconsistent.